chord_metadata_service/chord/workflows/wdls/document.wdl

```diff
@@ -0,0 +1,48 @@
+version 1.0
+
+workflow document {
+    input {
+        Array[File] document_files
+        String run_dir
+        String drs_url
+        String project_id
+        String dataset_id
+        String secret__access_token
+    }
+
+    scatter(file in document_files) {
+        call post_to_drs {
+            input:
+                file_path = file,
+                drs_url = drs_url,
+                project_id = project_id,
+                dataset_id = dataset_id,
+                token = secret__access_token
+        }
+    }
+
+    output {
+        Array[String] document_files_out = post_to_drs.response_message
```

Comment:
this is misnamed - it isn't file output, just responses - maybe call this `drs_responses` instead of `document_files_out`